consensus/src/consensus_observer/observer/consensus_observer.rs

```diff
@@ -837,8 +949,8 @@ impl ConsensusObserver {
                 Some(network_message) = consensus_observer_message_receiver.next() => {
```

Comment:
do we reject messages during fallback mode to avoid accumulating too many data?

Reply:
Yeah! We'll continue to pop them off the channel, and then drop them. This is because we unsubscribe from all peers before we fallback, and so when we do message sender verification, it will fail, e.g., here: https://github.com/aptos-labs/aptos-core/blob/9e4556c330a8bf6a82a4a2c43ba23b4defefcb53/consensus/src/consensus_observer/observer/consensus_observer.rs#L536